linear/linear.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+)
+
+func main(){
+	var a, b float64
+	fmt.Println("The program solves a linear inequality Ax + B > 0.Enter A and B: ")
+	fmt.Scan(&a, &b)
+	if a < 0 {
+		x := b / a
+		fmt.Println("x <", x)
+	} else if a == 0{
+		x := -b / a
```

Comment:
If `a == 0`, so `-b / a` is division by zero. Instead, you should check if `b > 0`, or not. If it is, then all real numbers are roots; otherwise, there are no roots at all.